Code/ObjectMapping/RKMappingErrors.h

```diff
@@ -20,8 +20,7 @@
 
 #import "RKErrors.h"
 
-typedef UInt32 RKMappingErrorCode;
-enum {
+typedef NS_ENUM(NSInteger, RKMappingErrorCode) {
```

Comment:
I believe `NSInteger` is much easier to standardize on, especially since it's the type of `NSError.code` which is what we're representing